doc/shared/sundials/Fortran.rst

```diff
@@ -18,11 +18,13 @@ Fortran Interface
 SUNDIALS provides modern, Fortran 2003 based, interfaces as Fortran modules to
 most of the C API including:
 
+- The SUNDIALS core types, utilities, and data structures via the ``fsundials_core_mod`` module.
+
 - All of the time-stepping modules in ARKODE:
 
   * The ``farkode_arkstep_mod``, ``farkode_erkstep_mod``, and
-    ``farkode_mristep_mod`` modules provide interfaces to the ARKStep, ERKStep,
-    and MRIStep integrators respectively.
+    ``farkode_mristep_mod``, ``farkode_sprkstep_mod`` modules provide interfaces
+   to the ARKStep, ERKStep, MRIStep, and SPRKStep integrators respectively.
```

Comment:
There is a misplaced "and" in the preceding line, that should be moved just before `farkode_sprkstep_mod`.